server/index.js

```diff
@@ -44,11 +46,47 @@ app.use(jsonMiddleware);
 
 app.use(staticMiddleware);
 
+app.use(cookieParser);
+
+app.post('/api/new-user', (req, res, next) => {
```

Comment:
Either `/api/users` or `/api/user` is fine since POST already tells us the action.  Otherwise we can merge.

Reply:
👍